src/components/TopBar/TopBar.vue

```diff
@@ -303,15 +303,13 @@ export default {
 <style lang="scss" scoped>
 .top-bar {
 	--border-width: 1px;
-	// hardcoded 1.5 value of line-height for compatibility with older versions
-	--text-height: calc(1.5 * (var(--default-font-size) + var(--font-size-small, 15px)));
 	display: flex;
 	flex-wrap: wrap;
 	z-index: 10;
 	gap: 3px;
 	align-items: center;
 	justify-content: flex-end;
-	min-height: calc(2 * var(--default-grid-baseline) + var(--text-height) + var(--border-width));
+	min-height: calc(var(--border-width) + 2 * (2 * var(--default-grid-baseline)) + var(--default-clickable-area));
```

Comment:
Call Container uses magic number 60 or close, need to be updated to the same value there too

Reply:
Done in a hotfix

Reply:
Ok, but I thought, we're going for variables)

Reply:
I found it more complex and risky. `CallView` is used in different entrypoints and not directly related to `TopBar`